contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1749,8 +1751,27 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		return;
 
 	type_def = defGetTypeName(as_type);
+	type_names = type_def->names;
+	switch (list_length(type_names))
+	{
+		case 2:
+			strVal(linitial(type_names)) = get_physical_schema_name(get_cur_db_name(),strVal(linitial(type_names)));
+			break;
+		case 3:
+			strVal(lsecond(type_names)) = get_physical_schema_name(strVal(linitial(type_names)),strVal(lsecond(type_names)));
+			linitial(type_names) = lsecond(type_names);
```

Comment:
instead of this two line, can't we just directly assign physical schema name to first element rather than assign it to second element first and then assign second element to first.

Reply:
Yes, that would be a better option. I have applied the change.